From patches Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
To: Paolo Bonzini <pbonzini@redhat.com>, qemu-devel <qemu-devel@nongnu.org>,
 Anthony Liguori <anthony@codemonkey.ws>, Hans de Goede <hdegoede@redhat.com>,
 None <fred.konrad@greensocs.com>, Markus Armbruster <armbru@redhat.com>,
 =?ISO-8859-1?Q?KONRAD_Fr=E9d=E9ric?= <fred.konrad@greensocs.com>,
 Gerd Hoffmann <kraxel@redhat.com>,
 KONRAD =?utf-8?B?RnLDqWTDqXJpYw==?= <fred.konrad@greensocs.com>
Cc: Paolo Bonzini <pbonzini@redhat.com>, qemu-devel <qemu-devel@nongnu.org>,
 =?UTF-8?B?S09OUkFEIEZyw6lkw6lyaWM=?= <fred.konrad@greensocs.com>,
 qemu-devel@nongnu.org, Hans de Goede <hdegoede@redhat.com>,
 None <fred.konrad@greensocs.com>,
 KONRAD =?utf-8?B?RnLDqWTDqXJpYw==?= <fred.konrad@greensocs.com>
Date: Tue,  9 Apr 2013 10:22:35 +0200
Message-id: 1365495755-10902-1-git-send-email-hdegoede@redhat.com
Subject: [PATCH 1/1] virtio-9p: Fix virtio-9p no longer building after hw-dirs
 branch merge

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Cc: Paolo Bonzini <pbonzini@redhat.com>
---
 hw/9pfs/virtio-9p-device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/9pfs/virtio-9p-device.c b/hw/9pfs/virtio-9p-device.c
index 43f930e..b476b81 100644
--- a/hw/9pfs/virtio-9p-device.c
+++ b/hw/9pfs/virtio-9p-device.c
@@ -12,11 +12,11 @@
  */
 
 #include "hw/virtio/virtio.h"
+#include "hw/virtio/virtio-9p.h"
 #include "hw/i386/pc.h"
 #include "qemu/sockets.h"
 #include "virtio-9p.h"
 #include "fsdev/qemu-fsdev.h"
-#include "virtio-9p-device.h"
 #include "virtio-9p-xattr.h"
 #include "virtio-9p-coth.h"
 
-- 
1.8.2


